src/components/behandlerdialog/BehandlerdialogPanel.tsx

```diff
@@ -0,0 +1,9 @@
+import styled from "styled-components";
+import Panel from "nav-frontend-paneler";
+
+export const BehandlerdialogPanel = styled(Panel)`
```

Comment:
Brukes denne kun på den nye siden? I så fall kan det vel bare ligge i samme fil som den brukes, siden det kun er styling?

Reply:
Jo, tenkte litt på det der 🤔 For når vi lager det andre panelet så vil det uansett legges i samme komponent 🤷🏼‍♂️ Så får vi heller flytte den ut hvis vi må bruke flere steder, I guess 👍🏼